src/components/Typography/BodyX/__snapshots__/BodyX.test.tsx.snap

```diff
@@ -196,29 +196,6 @@ Etiam at mi lacinia, gravida diam ac, sodales elit. Maecenas vitae rutrum nisl.
 Nunc pellentesque, arcu vitae bibendum rutrum, mi sapien aliquam ipsum, sit amet mollis quam orci quis est.
 Curabitur sagittis mauris volutpat lorem elementum, nec porttitor eros commodo. Suspendisse lorem orci, commodo mollis quam eget, facilisis tincidunt mauris.
 Ut in feugiat nunc, in eleifend ex. END
-    <span
```

Comment:
typography expansion does not exist anymore?

Reply:
Removed tests that verify props of Antd components

Reply:
reset test. I'm study why Ellipsis does not work